### ISSUE TITLE
Crash issue

### ISSUE THREAD

**Issue by FLORIANSV35** [psmokotnin/osm#55]
Hi ! 
i have some crash issue when i whant to load a session or make a new session.

I have change my os language from French to English (US) and it seems to be more stable.

i have this log : 
..\osmwin\src\audio\plugins\asioplugin.cpp(103):Debug: "OCTA-CAPTURE" ASIOInit
..\osmwin\src\generator\generatorthread.cpp(73):Debug: audio device not present  "OCTA-CAPTURE"
(0):Warning: QIODevice::write (InputDevice): device not open
(0):Warning: Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
(0):Warning: Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter

I'll try with the octa-capture plugged.

osm : V1.2.1
os : Windows 10 21H2
pc : Dell Latitude 5404

**Comment by FLORIANSV35** [comment]
Same thing with my octa-capture plugged

**Comment by psmokotnin** [comment]
Is it happens all the time you open or create a new measurement?

**Comment by DaGuus** [comment]
I have a similar issue on my framework laptop with windows 11 and 12th gen intel CPU

..\osmwin\src\audio\plugins\asioplugin.cpp(103):Debug: "X-USB ASIO Driver" ASIOInit
(0):Warning: QIODevice::write (InputDevice): device not open
(0):Warning: Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
(0):Warning: Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter
..\osmwin\src\generator\generatorthread.cpp(73):Debug: audio device not present  "{0.0.0.00000000}.{878fa5b2-c5d0-4a9c-a2bb-a09a9bd7b40a}"
..\osmwin\src\audio\plugins\asioplugin.cpp(103):Debug: "X-USB ASIO Driver" ASIOInit
(0):Warning: QIODevice::write (InputDevice): device not open
(0):Warning: Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
(0):Warning: Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter
..\osmwin\src\generator\generatorthread.cpp(73):Debug: audio device not present  "{0.0.0.00000000}.{878fa5b2-c5d0-4a9c-a2bb-a09a9bd7b40a}"
(0):Warning: Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
(0):Warning: Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter
..\osmwin\src\generator\generatorthread.cpp(73):Debug: audio device not present  "{0.0.0.00000000}.{878fa5b2-c5d0-4a9c-a2bb-a09a9bd7b40a}"
..\osmwin\src\audio\plugins\asioplugin.cpp(103):Debug: "X-USB ASIO Driver" ASIOInit
(0):Warning: QIODevice::write (InputDevice): device not open
(0):Warning: Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
(0):Warning: Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter

I get these errors with my X32 plugged in and not plugged in.

On my different laptop running windows 11 it works fine.

**Comment by daiptr** [comment]
any updates on this? i have the same. the app v.1.2.3 is crashing on startup:

```
audio device not present  "{0.0.0.00000000}.{ee222641-d6b5-4ef6-a4c4-ecddd0bb6548}"
QIODevice::write (InputDevice): device not open
Invalid QML element name "meta::Filter"; type names must begin with an uppercase letter
Invalid QML element name "meta::Measurement"; type names must begin with an uppercase letter
```

Lenovo thinkpad E460, win10pro 22H2. Conexant Audio Driver for Windows 10 (64-bit) - ThinkPad E460, E465, E560, E565

**Comment by psmokotnin** [comment]
There was an issue with the Windows Register, it's already solved. Fix will be included in v1.3.
